utils/managed_bytes.hh

```diff
@@ -479,36 +479,36 @@ std::invoke_result_t<Func, bytes_view> managed_bytes_view::with_linearized(Func&
 }
 
 // These are used to resolve ambiguities because managed_bytes_view can be converted to managed_bytes and vice versa
-inline bool operator==(const managed_bytes& a, const managed_bytes_view_base& b) {
+inline bool operator==(const managed_bytes& a, const managed_bytes_view_base& b) noexcept {
     return managed_bytes_view(a) == b;
```

Comment:
Let's leave these until later to reduce noise. They don't really add anything.

Reply:
ok